nixos/modules/security/wrappers/default.nix

```diff
@@ -227,7 +231,7 @@ in
   };
 
   ###### implementation
-  config = {
+  config = lib.mkIf config.security.enableWrappers {
```

Comment:
Should we perhaps treat this as an assertion rather than a toggle, failing if any wrappers would otherwise be generated, forcing people to use `lib.mkForce` to express that they definitely don’t need the wrappers that modules have set up for them? I’m okay doing it either way, but I wonder if this might not help track down potential problems and help ensure toggling the option doesn’t silently break the system.

Reply:
The primary motivation behind this change is to be able to disable them (e.g. via lib.mkDefault) in a config but then to be able to re-enable all of them at once without having to duplicate the values (e.g. via lib.mkForce for a e.g. a debug build). The assertion wouldn't really achieve that.

Reply:
Okay, makes sense to me. My personal motivation for something like this was to ensure that my system doesn’t depend on any setuid stuff, but that seems like a reasonable use‐case :)